test/unit/annotation_spec.js

```diff
@@ -37,6 +37,11 @@ describe("annotation", function () {
   class PDFManagerMock {
     constructor(params) {
       this.docBaseUrl = params.docBaseUrl || null;
+      this.pdfDocument = {
+        acroForm: () => {
+          return { get: x => undefined };
+        },
```

Comment:
The `acroForm` should contain an actual `Dict`, rather than a regular object. Hence please change this to: 
```suggestion
        acroForm: new Dict(),
```